shared/torngit/github.py

```diff
@@ -70,10 +69,44 @@ class Github(TorngitBaseAdapter):
         author="{username}/{name}/commits?author=%(author)s",
     )
 
+    @classmethod
+    def get_service_url(cls):
+        return get_config("github", "url", default="https://github.com").strip("/")
+
+    @property
+    def service_url(self):
+        return self.get_service_url()
+
+    @classmethod
+    def get_api_url(cls):
+        return get_config("github", "api_url", default="https://api.github.com").strip(
+            "/"
+        )
+
+    @classmethod
+    def get_api_host_header(cls):
+        return get_config("github", "api_host_override")
+
+    @classmethod
+    def get_host_header(cls):
+        return get_config("github", "host_override")
```

Comment:
You could do `get_config(cls.service, "host_override")` here and then remove the similar method in `github_enterprise.py`  (same for `get_api_host_header` above)

Reply:
Thanks for the tip. Added